docs/user/exercises/quiz.rst

```diff
@@ -1,2 +1,440 @@
+
+    - In the quiz exercises section click on the Create Quiz button |CreateDragAndDropQuizButton|.
+
+        .. figure:: quiz/apollon-diagrams.png
+            :align: center
+
+    - In the Apollon Diagrams page is possible to see the list of Apollon Diagrams and the possible actions to perform with them
+
+        - Open
+        - Delete
+
+    - Clicking in the creation of a new Apollon Diagram button |CreateANewApollonDiagram| opens the following form
+
+        .. figure:: quiz/Apollon-form.png
+            :align: center
+            :scale: 50
+
+        - Title: provide the title of the Drag and Drop Model Quiz
+
+        - Diagram Type: List picker that allows to select among several diagrams
+
+            - Class Diagram
+
+            - Activity Diagram
+
+            - Object Diagram
+
+            - Use Case Diagram
+
+            - Communication Diagram
+
+            - Component Diagram
+
+            - Deployment Diagram
+
+            - Petri Net
+
+            - Syntax Tree
+
+            - Flowchart
+
+        - Save Button
+
+        - Cancel Button
+
+    - Click the save button to open Apollon editor
+
+        .. figure:: quiz/apollon-editor.png
+            :align: center
+
+        - Title: Allows to edit the tile of the diagram.
+
+        - Crop image to selection: Allows to download the current selection.
+
+        - Download Button: To download the selection.
+
+        - Generate a Quiz Exercise Button.
+
+        - Save Button.
+
+        - Modeling field: The items for modeling the diagram will be displayed here.
+
+        - Elements to Drag and Drop in the Modeling field.
+
+Import a Quiz
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    - In the quiz exercises section, click on the Import a Quiz button |ImportQuizButton|.
+
+    - The list of existing quizzes will appear.
+
+        .. figure:: quiz/import-list-quizzes.png
+            :align: center
+            :scale: 50
+
+        - The searching bar: Allows to look for a specific quiz by tipping its name or part of it.
+
+        - The list of quizzes: Whit their ID, title, course and and indicator if they are exam questions.
+
+        - By clicking the Import Button |ImportButton| opens the quiz editor with the existing questions. Here is possible to edit all parameter such as in the **Create new quiz exercise**.
+
+Export a Quiz
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    - In the quiz exercises section, click on the Export Quiz Exercises button |ExportQuizExerciseButton| and the list of quizzes will be shown
+
+        .. figure:: quiz/export-quizzes-list.png
+            :align: center
+            :scale: 50
+
+        - Select te quizzes for being exported in the Export Column.
+
+        - The Export Button |ExportButton| will download the quiz in a JSON file.
+
+Resolve a quiz exercise
+--------
+
+    - In the Artemis Homepage, if there is a quiz exercise as a current exercise, will be possible to see it from the course overview or inside any course.
```

Comment:
```suggestion
    - If a quiz exercise is available on the Artemis Homepage as a current exercise, it will be possible to see it in the course overview or inside any course.
```

Can you really see a current quiz exercise in ANY course? Also if it belongs to another course?